3-platform/warehouse/db_postgres/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,7 @@
+databaseChangeLog:
```

Comment:
Is it asked to have liquibase in the scope of this task?

Reply:
Nope. Have deleted.

Reply:
I still see this file. Did you push changes?

Reply:
Not yet.